app/controllers/UploadFormController.scala

```diff
@@ -129,14 +130,14 @@ class UploadFormController @Inject() (
             case Some(Quarantined) =>
               Future.successful(Redirect(routes.VirusErrorController.onPageLoad()))
             case Some(Failed) =>
-              errorHandler.onServerError(request, new Throwable("Upload to upscan failed"))
+              renderer.render("serviceError.njk").map(Ok(_))
```

Comment:
@iservport serverError good to send it as status InternalServerError rather then OK